<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSlotGroupBox</class>
 <widget class="QGroupBox" name="CSlotGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>358</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Slot _:empty</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="startBlendLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Start blend</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="startBlendSpinBox">
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="endBlendLabel">
       <property name="text">
        <string>End blend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="endBlendSpinBox">
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="offsetLabel">
       <property name="text">
        <string>Offset frame</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="offsetSpinBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="speedLabel">
       <property name="text">
        <string>Speed x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="speedDoubleSpinBox">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="startFrameLabel">
       <property name="text">
        <string>Start frame</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="startFrameSpinBox">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="endFrameLabel">
       <property name="text">
        <string>End frame</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="endFrameSpinBox">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="smoothLabel">
       <property name="text">
        <string>Smooth</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="smoothSpinBox">
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="wrapModeLabel">
       <property name="text">
        <string>Wrap mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="clampRadioButton">
       <property name="text">
        <string>Clamp</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="repeatRadioButton">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="disableRadioButton">
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="animPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set anim</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skelPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set skel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="alignBlendPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Align blend</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="enableCheckBox">
       <property name="text">
        <string>Enable</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="invSkelWeightCheckBox">
       <property name="text">
        <string>Invert skeleton weight</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
